scripts/upload_incr_DB.py

```diff
@@ -245,6 +252,31 @@ def GetAndUploadStateDeltaDiff(blockNum, lastBlockNum):
 		return 0 #success
 	return 1
 
+def GetStaticFoldersFromS3(url, folderName):
+    list_of_folders = []
+    MARKER = ""
+    # Try get the entire persistence keys.
+    # S3 limitation to get only max 1000 keys. so work around using marker.
+    while True:
+        response = requests.get(url, params={"prefix":folderName, "max-keys":1000, "marker":MARKER, "delimiter":"/"})
+        tree = ET.fromstring(response.text)
+        if(tree[6:] == []):
+            print("Empty response")
+            break
+        lastkey = ''
+        for key in tree[6:]:
+            key_url = key[0].text.split(folderName,1)[1].replace('/', '')
+            if key_url != '':
+                list_of_folders.append(key_url)
+            lastkey = key_url
```

Comment:
> should statement on line 271 ,be kept within if statement scope ?
In fact this should actually be `lastkey = key[0].text` because `key_url` is just a part of the key. Thanks for pointing it out.